addon/components/nrg-home-cards/home-card/component.js

```diff
@@ -15,21 +15,28 @@ export default Component.extend({
   router: service(),
 
   visible: computed('currentUser.roles', function() {
-    const role = this.get('role');
+    const role = this.role;
```

Comment:
I think it might be weird to have `role` and `roles`. Thoughts?

Reply:
A little. I wanted to do it in a way that didn't break existing functionality. I suppose I could do that by checking to see if `role` is an array if that sounds good.

Reply:
What about normalizing to array instead?